src/lib.rs

```diff
@@ -103,7 +103,7 @@ pub mod receive_imf;
 pub mod tools;
 
 pub mod accounts;
-
+mod utils;
```

Comment:
BTW there is already a module `tools` which to my understanding means the same as `utils`, even though I agree that `utils` would be a slightly better name

Reply:
oh I thought I fixed that but thx anyway